### PR TITLE
Use legacy pass manager when wasm EH is enabled

### PR DESCRIPTION
**Pull request by aheejin**
This is a temporary measure for our clients who are experimenting with
this new feature. See #14180.

Not sure how to add a test for this.

### CODE REVIEW

File: emcc.py [hunk 1]
```diff
@@ -856,6 +856,11 @@ def get_cflags(options, user_args):
              '-D__unix',
              '-D__unix__']
 
+  # When wasm EH is enabled, we use the legacy pass manager because the new pass
+  # manager + wasm EH has some known bugs. TODO Use the new pass manager.
```
Comment (dschuff) [inline emcc.py:860]: when adding TODOs tracked by bugs, it's a good idea to add a link to the bug along with the todo.

**Review by dschuff — APPROVED** [review]

**Comment by dschuff** [comment]
Yeah, If if we don't have a small reproducer, I'm not sure of a good way to test either.

**Review by kripken — APPROVED** [review]
(I agree there's no need for a test.)

File: emcc.py (hunk 1 above)
Comment (aheejin) [reply emcc.py]: Done

**Pull request merged**